system/jlib/jsocket.cpp

```diff
@@ -1579,7 +1581,7 @@ void CSocket::read(void* buf, size32_t min_size, size32_t max_size, size32_t &si
 {
     unsigned startt=usTick();
     size_read = 0;
-    unsigned start;
+    unsigned start = 0;
```

Comment:
Took this opportunity and added initialization only to prevent a compile warning.